man/froll.Rd

```diff
@@ -36,6 +37,7 @@
   \item{na.rm}{ Logical, default \code{FALSE}. Should missing values be removed when calculating window? }
   \item{hasNA}{ Logical. If it is known that \code{x} contains \code{NA} (or \code{Nan}) then setting this to \code{TRUE} will speed up calculation. Defaults to \code{NA}. See \emph{hasNA argument} section below for details. }
   \item{adaptive}{ Logical, default \code{FALSE}. Should the rolling function be calculated adaptively? See \emph{Adaptive rolling functions} section below for details. }
+  \item{partial}{ Logical, default \code{FALSE}. Should the rolling window size(s) provided in \code{n} be trimmed to available observations. See \emph{\code{partial} argument} section below for details. }
```

Comment:
I worry "available observations" has some overlap with the concept of "missingness" and might be confusing.

```suggestion
  \item{partial}{ Logical, default \code{FALSE}. Should the rolling window size(s) provided in \code{n} be trimmed to exclude indices outside the vector? See \emph{\code{partial} argument} section below for details. }
```

Reply:
Doesn't sound any more clear to me...

Reply:
"trimmed not to extend outside the vector" maybe?

Reply:
I made it even more simple, as we point reader to detailed section below.